myCUinfo.py

```diff
@@ -173,13 +171,20 @@ def info(self):
         # set the college (Arts and Scineces.. ect)
         info["college"] = splitText[10 + employeeShift].strip()[16:-18]
 
+        #For some reason some categories have two indexes, one of which is empty
+
         # set their major
         info["major"] = splitText[11 + employeeShift].strip()[14:-15]
+        if info["major"]=="":
+            info["major"] = splitText[17 + employeeShift].strip()[14:-15]
+        if info["major"]=="":
+            info["major"]=None
 
         # if they have a minor, set the minor
-        if splitText[13].strip()[16:-17] != "":
-            info["minor"] = splitText[13].strip()[16:-17]
-        else:
+        info["minor"] = splitText[12 + employeeShift].strip()[16:-17]
+        if info["minor"] == "":
+            info["minor"] = splitText[18 + employeeShift].strip()[16:-17]
```

Comment:
I might just change it to iterate the splitText array for data since idk if those values can be hard coded anymore. I will change it after I fix classes()